src/public/class-wordlift-context-cards.php

```diff
@@ -28,81 +30,126 @@ function __construct() {
-			if( isset($jsonld[0]['name']) && !empty($jsonld[0]['name']) ) {
-				$title = $jsonld[0]['name'];
-				$pos = strpos($jsonld[0]['description'], $title);
+		if ( isset( $jsonld[0]['description'] ) && ! empty( $jsonld[0]['description'] ) ) {
+			if ( isset( $jsonld[0]['name'] ) && ! empty( $jsonld[0]['name'] ) ) {
+				$title                   = $jsonld[0]['name'];
+				$pos                     = strpos( $jsonld[0]['description'], $title );
 				$response['description'] = $jsonld[0]['description'];
-				if ($pos !== false) {
-					$response['description'] = substr_replace($response['description'], "<strong>$title</strong>", $pos, strlen($title));
+				if ( $pos !== false ) {
+					$response['description'] = substr_replace( $response['description'], "<strong>$title</strong>", $pos, strlen( $title ) );
 				}
 			} else {
 				$response['description'] = $jsonld[0]['description'];
 			}
 		}
 
-		if( isset($jsonld[0]['name']) && !empty($jsonld[0]['name']) ){
+		if ( isset( $jsonld[0]['name'] ) && ! empty( $jsonld[0]['name'] ) ) {
 			$response['title'] = $jsonld[0]['name'];
 		}
 
-		if( isset($jsonld[0]['url']) && !empty($jsonld[0]['url']) ){
+		if ( isset( $jsonld[0]['url'] ) && ! empty( $jsonld[0]['url'] ) ) {
 			$response['url'] = $jsonld[0]['url'];
 		}
 
-		if( isset($jsonld[0]['image']) &&
-		    isset($jsonld[0]['image'][0]['url']) &&
-		    isset($jsonld[0]['image'][0]['width']) &&
-		    isset($jsonld[0]['image'][0]['height'])
-		){
+		if ( isset( $jsonld[0]['image'] ) &&
+		     isset( $jsonld[0]['image'][0]['url'] ) &&
+		     isset( $jsonld[0]['image'][0]['width'] ) &&
+		     isset( $jsonld[0]['image'][0]['height'] )
+		) {
 			$response['image'] = array(
-				'url' => $jsonld[0]['image'][0]['url'],
-				'width' => $jsonld[0]['image'][0]['width'],
+				'url'    => $jsonld[0]['image'][0]['url'],
+				'width'  => $jsonld[0]['image'][0]['width'],
 				'height' => $jsonld[0]['image'][0]['height'],
 			);
 		}
 
-		if($publisher){
-			$publisher_id       = Wordlift_Configuration_Service::get_instance()->get_publisher_id();
-			$publisher_jsonld   = Wordlift_Jsonld_Service::get_instance()->get_jsonld( false, $publisher_id );
+		if ( $publisher ) {
+			$publisher_id          = Wordlift_Configuration_Service::get_instance()->get_publisher_id();
+			$publisher_jsonld      = Wordlift_Jsonld_Service::get_instance()->get_jsonld( false, $publisher_id );
 			$response['publisher'] = $this->format_response( $publisher_jsonld, false );
 		}
 
 		return $response;
 	}
 
+	function format_as_jsonld( $jsonld, $publisher = true ) {
+
+		$response = array();
+
+		if ( ! isset( $jsonld ) || empty( $jsonld ) || empty( $jsonld[0] ) ) {
+			return null;
+		}
+
+		$publisher_id     = Wordlift_Configuration_Service::get_instance()->get_publisher_id();
+		$publisher_jsonld = Wordlift_Jsonld_Service::get_instance()->get_jsonld( false, $publisher_id );
+
+		$response['@context']  = "http://schema.org";
```

Comment:
@akshayraje use `Wordlift_Jsonld_Service::get_instance()->get_jsonld( true );` to get the WebSite markup.